Error while building project

Hi, I just tried to build the project with `make windows` but it fails with below error. Also getting the same error on Ubuntu 20.04
```
npm install

up to date, audited 465 packages in 1s

55 packages are looking for funding
  run `npm fund` for details

3 vulnerabilities (2 moderate, 1 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
curl -L  -o .zip
curl: (2) no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
mingw32-make: *** [Makefile:22: fonts/NORMAL/.zip] Error 2
```

--- a/main/setting.ts
+++ b/main/setting.ts
@@ -10,7 +10,7 @@ export class Setting {
   private static path: string = join(app.getPath("appData"), "envim.json");
 
   static set(item: ISetting) {
-    const presets = Setting.item.presets || {};
+    const { presets } = Setting.item || { presets: {} };
 
     Setting.init = true;
     Setting.item = { ...item, presets };

